test/test-step-03.bash

```diff
@@ -13,4 +13,20 @@ KCOV_DIR=_kcov03 \
 # Verify we can build a commit that is not on the main branch
 SKIP_DOCKER_BUILD=1 \
 KCOV_DIR=_kcov03 \
-../enki --keep-data --data-dir $BOOK_DIR --command all-git-web --repo 'philschatz/tiny-book' --book-slug 'book-slug1' --style chemistry --ref @458dfb710e9af3d00d6f7e0be45fc819b955d931 --start-at git-disassemble
\ No newline at end of file
+../enki --keep-data --data-dir $BOOK_DIR --command all-git-web --repo 'philschatz/tiny-book' --book-slug 'book-slug1' --style chemistry --ref @458dfb710e9af3d00d6f7e0be45fc819b955d931 --start-at git-disassemble
+
+
+# Check local-preview works
+SKIP_DOCKER_BUILD=1 \
+KCOV_DIR=_kcov03 \
+../enki --keep-data --data-dir $BOOK_DIR --command local-preview
+
+# Check local-preview directories / files existing
+LOCAL_PREVIEW="$BOOK_DIR/local-preview"
+[ -d "$LOCAL_PREVIEW" ] || echo "local-preview: Directory $LOCAL_PREVIEW missing"
```

Comment:
Should there be a non-zero exit code if one of these conditions is not met?

Reply:
oh, big mistake here 🤦‍♂️ will fix